### PR TITLE
Clean up CLI output a bit.

### PR DESCRIPTION
**Pull request by robdodson**
This improves #3605 a little bit but doesn't fully fix the problem.

I think after [Eleventy's `afterBuild` events ship in 1.0](https://www.11ty.dev/docs/events/) we should be able to use https://www.npmjs.com/package/cli-spinner to show when a build is in progress or finished.

### CODE REVIEW

File: src/site/_filters/find-by-url.js [hunk 1]
```diff
@@ -33,11 +30,6 @@ let memo;
  * @see {@link https://github.com/11ty/eleventy/issues/399}
  */
 const memoize = (collection) => {
```
Comment (robdodson) [inline src/site/_filters/find-by-url.js:32]: The warning that was previously here was essentially noise (it logged every time you edited a file in dev mode) so I just removed it.

File: .eleventy.js [hunk 2]
```diff
@@ -91,6 +92,7 @@ const {
 } = require(`./${transformsDir}/service-worker-partials`);
 
 module.exports = function (config) {
+  console.log(`${chalk.black.bgGreen('Eleventy is building, please wait…')}`);
```
Comment (samthor) [inline .eleventy.js:95]: This only happens once, right?

Also, super nit, the `${` doesn't really help here, we could just remove the wrapping.

Comment (samthor) [reply .eleventy.js:95]: Also, not sure if you're a fan, but we use `fancy-log` in a few place which includes timestamp on the left. We could use it here too.

Comment (robdodson) [reply .eleventy.js:95]: Yeah, that's why it doesn't help a ton. It happens when you first run `npm run dev` but if you subsequently edit a file it won't log it again.

> Also, super nit, the ${ doesn't really help here, we could just remove the wrapping.
oh right, I was combining strings earlier and then put it all into the same message. I'll fix it.

**Pull request merged**